cmd/root.go

```diff
@@ -248,7 +248,8 @@ func init() {
 
 	RootCmd.PersistentFlags().String("logs-level", "Info", "Logs level. Supported log levels are Trace, Debug, Info, Warning, Off. If the log level is set to Off, Atmos will not log any messages")
 	RootCmd.PersistentFlags().String("logs-file", "/dev/stderr", "The file to write Atmos logs to. Logs can be written to any file or any standard file descriptor, including '/dev/stdout', '/dev/stderr' and '/dev/null'")
-
+	RootCmd.PersistentFlags().String("base-path", "", "Base path for Atmos configuration files.")
```

Comment:
```suggestion
	RootCmd.PersistentFlags().String("base-path", "", "Base path for Atmos project")
```

Reply:
"configuration files" is confusing, because "Atmos configuration" could mean the atmos.yaml. This is more for components and stacks.